experimental/PropertyDDS/packages/property-query/package.json

```diff
@@ -45,6 +45,7 @@
 		"@fluid-internal/mocha-test-setup": "workspace:~",
 		"@types/sinon": "^17.0.3",
 		"chai": "^4.2.0",
+		"chai-as-promised": "7.1.1",
```

Comment:
The property-query package depends on these two dependencies that were previously only installed by property-query-service.  property-query was able to reach these since we don't ban hoisting in the repo (but maybe we should).